src/examples/src/widgets/menu/LargeOptionSet.tsx

```diff
@@ -0,0 +1,27 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Menu, { MenuOption } from '@dojo/widgets/menu';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+
+export default factory(function LargeOptionSet({ middleware: { icache } }) {
+	const options: MenuOption[] = [];
+	const total = 100000;
+	for (let i = 0; i < total; i++) {
```

Comment:
I don't think you need to do this every render?

Reply:
fair